source/features/rgh-feature-descriptions.tsx

```diff
@@ -24,7 +24,7 @@ function addDescription(infoBanner: HTMLElement, id: string, meta: FeatureMeta |
 			? 'This feature applies only to "Refined GitHub" repositories and cannot be disabled'
 			: (
 				isCss
-					? 'This feature is CSS-only cannot be disabled'
+					? 'This feature is CSS-only and cannot be disabled'
```

Comment:
Should the messages on this line and line 24 end with a period, as all of the descriptions in the readme end with one?

```suggestion
					? 'This feature is CSS-only and cannot be disabled.'
```